modulemd/v2/include/modulemd-2.0/private/modulemd-module-private.h

```diff
@@ -96,7 +96,10 @@ modulemd_module_get_translation (ModulemdModule *self, const gchar *stream);
 /**
  * modulemd_module_add_stream:
  * @self: This #ModulemdModule object
- * @stream: A #ModulemdModuleStream object to associate with this #ModulemdModule.
+ * @stream: A #ModulemdModuleStream object to associate with this
+ * #ModulemdModule. A stream added to a #ModulemdModule must have a module
```

Comment:
Maybe here mention that it will be ignored? Since there's no error feedback other than the logged output.